src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/ManageEvidenceNotesControllerReviewEvidenceNoteTests.cs

```diff
@@ -0,0 +1,334 @@
+            var organisationId = model.ViewEvidenceNoteViewModel.OrganisationId;
+            var organisationName = Faker.Company.Name();
+            A.CallTo(() => Cache.FetchOrganisationName(organisationId)).Returns(organisationName);
+            EvidenceNoteData note = Fixture.Create<EvidenceNoteData>();
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>._)).Returns(note);
+            AddModelError();
+
+            // act
+            await ManageEvidenceController.ReviewEvidenceNote(model);
+
+            // asset
+            Breadcrumb.ExternalActivity.Should().Be(BreadCrumbConstant.SchemeManageEvidence);
+            Breadcrumb.ExternalOrganisation.Should().Be(organisationName);
+            Breadcrumb.OrganisationId.Should().Be(organisationId);
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNotePost_GivenInvalidModel_ModelShouldBeReturned()
+        {
+            //arrange
+            var httpContext = new HttpContextMocker();
+            var model = Fixture.Create<ReviewEvidenceNoteViewModel>();
+            AddModelError();
+            EvidenceNoteData note = Fixture.Create<EvidenceNoteData>();
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>._)).Returns(note);
+            httpContext.AttachToController(ManageEvidenceController);
+
+            //act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(model) as ViewResult;
+
+            //assert
+            result.Model.Should().NotBeNull();
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNotePost_GivenAModelIsValid_ShouldRedirectToDownloadEvidenceNoteAction()
+        {
+            //arrange
+            var httpContext = new HttpContextMocker();
+            httpContext.AttachToController(ManageEvidenceController);
+            var model = GetValidModel();
+
+            //act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(model) as RedirectToRouteResult;
+
+            // assert
+            result.RouteValues["action"].Should().Be("DownloadEvidenceNote");
+        }
+
+        [Fact]
+        public async Task ReviewEvidenceNotePost_GivenInvalidModel_ShouldRedirectToReviewEvidenceNoteAction()
+        {
+            //arrange
+            var httpContext = new HttpContextMocker();
+            var model = Fixture.Create<ReviewEvidenceNoteViewModel>();
+            AddModelError();
+            EvidenceNoteData note = Fixture.Create<EvidenceNoteData>();
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>._)).Returns(note);
+            httpContext.AttachToController(ManageEvidenceController);
+
+            //act
+            var result = await ManageEvidenceController.ReviewEvidenceNote(model) as ViewResult;
+
+            // assert
+            result.ViewName.Should().Be("ReviewEvidenceNote");
+        }
```

Comment:
Need a test to check that when the model is valid that the TempData[ViewDataConstant.EvidenceNoteStatus] is populated.